virl2_client/models/lab.py

```diff
@@ -332,6 +337,71 @@ def description(self, value: str) -> None:
         """Set the description of the lab."""
         self._set_property("description", value)
 
+    @property
+    def autostart_enabled(self) -> bool:
+        """Return whether autostart is enabled for the lab."""
+        self.sync_topology_if_outdated()
+        return self._autostart["enabled"]
+
+    @autostart_enabled.setter
+    def autostart_enabled(self, value: bool) -> None:
+        """Set whether autostart is enabled for the lab."""
+        self._autostart["enabled"] = value
+        self._set_property("autostart", self._autostart)
+
+    @property
+    def autostart_priority(self) -> int | None:
+        """Return the autostart priority of the lab."""
+        self.sync_topology_if_outdated()
+        return self._autostart["priority"]
+
+    @autostart_priority.setter
+    def autostart_priority(self, value: int | None) -> None:
+        """Set the autostart priority of the lab."""
+        if value is not None and not (0 <= value <= 10000):
+            raise ValueError("autostart_priority must be between 0 and 10000")
+        self._autostart["priority"] = value
+        self._set_property("autostart", self._autostart)
+
+    @property
+    def autostart_delay(self) -> int | None:
+        """Return the autostart delay of the lab."""
+        self.sync_topology_if_outdated()
+        return self._autostart["delay"]
+
+    @autostart_delay.setter
+    def autostart_delay(self, value: int | None) -> None:
+        """Set the autostart delay of the lab."""
+        if value is not None and (value < 0 or value > 84600):
```

Comment:
I like this better, but the original is fine too except that it's 86400
```suggestion
        if not (value is None or 0 <= value <= 86400):
```

Reply:
@marpauli-cisco 84600 -> 86400 needs to be changed in API and UI as well